### ISSUE TITLE
Building the project, the make file, and usage of inkscape.

### ISSUE THREAD

**Issue by jared-w** [hmemcpy/milewski-ctfp-pdf#72]
First things first, since you have a Makefile, you probably should just be calling the raw LaTeX commands to compile the program rather than using latexmk since it internally operates like its own build system and isn't really designed to be used inside other build systems.

Secondly, I have no idea why inkscape is required and used in the makefile. I looked at it and it appears that inkscape is only used to embed a *.svg into the pdf to use it as the cover page? If that's the case, inkscape can be used to output LaTeX itself and then after that you can just compile all the LaTeX files together using TeX itself with no need for an external program.

I'm mostly writing this issue as a reminder for myself to take a look at this and see if I can clean the makefile up and remove the inkscape dependency. A simple `make` should ideally be sufficient.

**Comment by hmemcpy** [comment]
Thanks for your comments. The simple answer to most of them: the makefile isn't mine :) I forked [another project](https://github.com/sarabander/sicp-pdf) and used their tools/preamble as basis of the book. Theirs is a whole lot more complicated, so I'm removing things as I go... also, I've never used LaTeX before this, so it's all very new to me.

You're right, inkscape is (yet another) step that could be simplified, as well as trimming the preamble - it has things I don't really need/use.

Please feel free to send a PR improving whatever you feel necessary. That's how we'll get the best possible result! 👍

**Comment by hmemcpy** [comment]
(p.s. for my own setup, I use [entr](entrproject.org) to re-run make whenever anything changes, so I can just edit the file(s) in VS Code, and it will be re-compiled on save.)

**Comment by jared-w** [comment]
Hmm, interesting; apparently using latexmk in the make file is pulling my environment's `~/.latexmkrc` file and messing things up when I make the pdf occasionally. I was able to fix this by passing -norc to the make file. I also customized the makefile to never remove the coverpage.pdf so now we never need to generate it with inkscape and I was able to remove that bit out of the makefile as well.

I also went through and attempted to cut out as many things from the preamble as I could; I commented the preamble with everything I thought we could probably get rid of. I also made a minor change to fix the parts in the table of contents. I'll open up a pull-request just as is so that it can be commented and discussed.
